frontend/test/metabase/scenarios/collections/collections-sidebar.cy.spec.js

```diff
@@ -0,0 +1,41 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("collections sidebar (metabase#15006)", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.visit("/collection/root");
+  });
+
+  describe("in desktop form factors", () => {
+    it("should not be able to toggle collections sidebar visibility", () => {
```

Comment:
@nemanjaglumac 

[nit]

Initially I went with exactly the same label you wrote on your example.

I ended up thinking "mobile sidebar" is not ideal, because the sidebar is written for both desktop and mobile.